src/odemis/acq/align/fastem.py

```diff
@@ -233,9 +240,11 @@ def run(self):
 
             # loop over calibrations in list (order in list is important!)
             for calib in self.calibrations:
-                logging.debug("Starting calibration %s", calib)
+                calib_cls = calib.value
+                logging.debug("Starting calibration %s", calib_cls.__name__)
+                calib_cls = calib_cls(components)
```

Comment:
It's not really nice to override a class by an instance. Can you store the instance in a different variable? Like `calib_runner` (but a better name!) .